Documentation/certificate-generation-resources.md

```diff
@@ -19,8 +19,8 @@ Parameters are passed to/from the endpoints as JSON key-value pairs.
 
 ## Settings on Bridge initialization
 
-* `BridgeCertificatePassword` - string - password for certificates generated/exported. Default is "test"
-* `BridgeCertificateValidityPeriod` - TimeSpan - valid timespan for certificates generated. Default is 24 hours
+* `TestRootCertificatePassword` - string - password for certificates generated/exported. Default is "test"
```

Comment:
I don't think these properties are actually used except in classes that themselves are not used.  Unless I'm missing something. I believe we should not bother changing them and nuke the files that use them in a separate PR.  See other comments for specifics.